cli/main.rs

```diff
@@ -445,15 +445,42 @@ async fn bundle_command(
     let output_bytes = output_string.as_bytes();
     let output_len = output_bytes.len();
     deno_fs::write_file(out_file_, output_bytes, 0o666)?;
-    // TODO(bartlomieju): add "humanFileSize" method
-    info!("{} bytes emitted.", output_len);
+    info!("{} emitted.", human_size(output_len as f64));
   } else {
     println!("{}", output_string);
   }
-
   Ok(())
 }
 
+fn human_size(bytse: f64) -> String {
+  let negative = if bytse.is_sign_positive() { "" } else { "-" };
+  let bytse = bytse.abs();
+  let units = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
+  if bytse < 1_f64 {
+    return format!("{}{} {}", negative, bytse, "Bytes");
+  }
+  let delimiter = 1000_f64;
+  let exponent = std::cmp::min(
+    (bytse.ln() / delimiter.ln()).floor() as i32,
+    (units.len() - 1) as i32,
+  );
+  let pretty_bytes = format!("{:.2}", bytse / delimiter.powi(exponent))
+    .parse::<f64>()
+    .unwrap()
+    * 1_f64;
+  let unit = units[exponent as usize];
+  format!("{}{} {}", negative, pretty_bytes, unit)
+}
+
+#[test]
+fn human_size_test() {
+  let units = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
+  for (index, unit) in units.iter().enumerate() {
+    let power = index as f64 * 3.0;
+    assert_eq!(human_size(10_f64.powf(power)), format!("{} {}", 1, unit))
+  }
```

Comment:
It would be better to have some explicit examples rather than abstract code. Example:
```rust
assert_eq!(human_size(16*1024*1024), "16 MB");
```